podbuilder/podbuilder.go

```diff
@@ -310,6 +310,14 @@ func (pb *PodBuilder) newPod() *corev1.Pod {
 		})
 	}
 
+	pb.run.Env = append(pb.run.Env, corev1.EnvVar{
+		Name:  "POD_TIMEOUT",
```

Comment:
nit: The name of this environment variable should be a global, like config.PodTimeoutEnv, to match the rest of the environment variables.